src/react/features/content/content.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Content
+
+  Rule: Show and control content in background
+
+  Background:
+    Given I am at home
+    And I take 'Generic content' in focus
+
+  Scenario: Play and pause video
+    When I take 'some-video.mov' in focus
+    Then I see pause button
+    But I do not see play button
+    When I select pause button
+    Then I do not see video playing
+    And I see play button
+    When I select play button
+    Then I see video playing
```

Comment:
Noissa muissa on tässä kohtaa, että "Navigated back and I don't see play button", jos haluaa olla yhdenmukainen